fingerprint_pro_server_api_sdk/__init__.py

```diff
@@ -127,6 +128,7 @@
 from fingerprint_pro_server_api_sdk.models.webhook_raw_device_attributes import WebhookRawDeviceAttributes
 from fingerprint_pro_server_api_sdk.models.webhook_remote_control import WebhookRemoteControl
 from fingerprint_pro_server_api_sdk.models.webhook_root_apps import WebhookRootApps
+from fingerprint_pro_server_api_sdk.models.webhook_supplementary_i_ds import WebhookSupplementaryIDs
```

Comment:
It might be better to name this module `fingerprint_pro_server_api_sdk.models.webhook_supplementary_ids` instead of `fingerprint_pro_server_api_sdk.models.webhook_supplementary_i_ds`. I'll check if that’s possible with the current codegen.